pkg/client/outofband/client.go

```diff
@@ -35,6 +35,18 @@ const (
 	InvitationMsgType = outofband.InvitationMsgType
 )
 
+// EventOptions are is a container of options that you can pass to an event's
+// Continue function to customize the reaction to incoming out-of-band messages.
+type EventOptions struct {
+	// Label will be shared with the other agent during the subsequent did-exchange.
```

Comment:
Do you have thoughts about this coupling? We don't know that the subsequent action will be a DID exchange right?

Reply:
It can be any protocol that forms a connection, really. Or it can be an ephemeral mode.

Reply:
So is Label intended to be generic or about did-exchange?

I would like to avoid this struct becoming a collection of all possible followup options. In that case, it seems better to pass options directly into the followup protocol implementation.